httpbin/core.py

```diff
@@ -636,16 +637,13 @@ def redirect_to():
     args_dict = request.args.items()
     args = CaseInsensitiveDict(args_dict)
 
-    # We need to build the response manually and convert to UTF-8 to prevent
-    # werkzeug from "fixing" the URL. This endpoint should set the Location
-    # header to the exact string supplied.
     response = app.make_response("")
     response.status_code = 302
     if "status_code" in args:
         status_code = int(args["status_code"])
         if status_code >= 300 and status_code < 400:
             response.status_code = status_code
-    response.headers["Location"] = args["url"].encode("utf-8")
```

Comment:
Was this issue resolved in a later version of werkzeug?  I am curious to understand why it is no longer necessary.

Reply:
I think that's covered in item 3 in the PR overview. Was there something more specific you were wondering about?